commons/tool/src/webapp/js/commons.js

```diff
@@ -124,6 +124,22 @@ commons.switchState = function (state, arg) {
                     editorCancelButton.prop('disabled', false);
                 }
                 editor.focus();
+            }).on('drop', function (e) {
+                // clear placeholder text
+                if (this.innerHTML == commons.i18n['post_editor_initial_text']) {
+                    this.innerHTML = '';
+                    $('#commons-editor-post-button').prop('disabled', false);
+                    editorPostButton.prop('disabled', false);
```

Comment:
These are the same thing, I think.

Reply:
Are you saying it's the same as the "click" part from line 119 above?  If so, yes, you are correct.  However, the click event doesn't fire when you drag and drop content, so I needed to do the same exact thing for the drop event.

Reply:
Oh, maybe you mean line 131 and 132 are doing the same thing?